frontend/src/layout/navigation-3000/sidepanel/panels/discussion/SidePanelDiscussion.tsx

```diff
@@ -54,14 +55,19 @@ export const SidePanelDiscussion = (): JSX.Element => {
         <div className="flex flex-col overflow-hidden flex-1">
             <SidePanelPaneHeader
                 title={
-                    <>
-                        Discussion{' '}
-                        {scope ? (
-                            <span className="font-normal text-muted-alt">
-                                about {item_id ? 'this' : ''} {humanizeScope(scope, !!item_id)}
-                            </span>
-                        ) : null}
-                    </>
+                    <div className="flex space-x-2">
+                        <span>
+                            Discussion{' '}
+                            {scope ? (
+                                <span className="font-normal text-muted-alt">
+                                    about {item_id ? 'this' : ''} {humanizeScope(scope, !!item_id)}
+                                </span>
+                            ) : null}
+                        </span>
+                        <Tooltip title="We may choose to remove Discussions at some point in the future">
```

Comment:
I like it but maybe less threatening 😅 
```suggestion
                        <Tooltip title="This is a feature we are experimenting with! We'd love to get your feedback on it and whether this is something useful for working with PostHog.">
```